datasets/ONS-International-trade-in-services-by-subnational-areas-of-the-UK/codelists/industry-grouping-mappings.csv

```diff
@@ -0,0 +1,22 @@
+Industry Grouping,SIC07 Mapping
+primary-utilities,A
+primary-utilities,B
+primary-utilities,D
+primary-utilities,E
+manufacturing,C
+construction,F
+wholesale-motor-trade,45
```

Comment:
The fact that there are string + integer values in here does cause a validation warning, but I'm not sure how I'd suppress that.